SpotifySearch/SpotifySearch/DetailViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DetailViewController.swift
+//  SpotifySearch
+//
+//  Created by Thinley Dorjee on 10/28/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+
+    @IBOutlet weak var DetialImage: UIImageView!
+    
+    var detailImage: Spotify!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        SpotifyAPIFactory.manager.downloadImage(urlString: detailImage.realImage) { (returnedData: Data) in
+            DispatchQueue.main.async {
+                self.detailImage.image = UIImage(data: returnedData)
```

Comment:
Watch how you name variables. Your detail imageView is called `DetialImage`. It's not an image, it's an image view. You name your Spotify object `detailImage`.

Can you see how that's confusing?